drivers/android-driver/src/main/java/com/squareup/sqldelight/android/AndroidSqliteDriver.kt

```diff
@@ -37,19 +37,22 @@ class AndroidSqliteDriver private constructor(
   /**
    * @param [cacheSize] The number of compiled sqlite statements to keep in memory per connection.
    *   Defaults to 20.
+   * @param [useNoBackupDirectory] Sets whether to use a no backup directory or not.
    */
   @JvmOverloads constructor(
     schema: SqlDriver.Schema,
     context: Context,
     name: String? = null,
     factory: SupportSQLiteOpenHelper.Factory = FrameworkSQLiteOpenHelperFactory(),
     callback: SupportSQLiteOpenHelper.Callback = AndroidSqliteDriver.Callback(schema),
-    cacheSize: Int = DEFAULT_CACHE_SIZE
+    cacheSize: Int = DEFAULT_CACHE_SIZE,
+    useNoBackupDirectory: Boolean = false
```

Comment:
This is a weirdly-named parameter. The builder method is also weird. They should remove the "no" and invert the boolean value.

Reply:
I agree. My gut feeling was to keep it for consistency with the builder, but can invert if preferable.

Reply:
I guess it's so named because the feature is called "no-backup".